src/test/shell/bazel/BUILD

```diff
@@ -236,6 +236,56 @@ sh_test(
     ],
 )
 
+sh_test(
+    name = "bazel_java_test_local_java_tools_jdk9",
+    size = "large",
+    timeout = "eternal",
```

Comment:
nit: why eternal? "large" doesn't cut it?

Reply:
I copy/pasted the test. I recently added a TODO (https://github.com/bazelbuild/bazel/blob/master/src/test/shell/bazel/BUILD#L151) to investigate if these fields still apply.